### ISSUE TITLE
Location Offer November 2024

### ISSUE THREAD

**Issue by McRip** [rughh/planning#178]
Thank you for wanting to host the Ruby Usergroup Hamburg!

Everything that there is to know about hosting you will in find in our [README](https://github.com/rughh/planning#ausrichten--hosting) and on [our website](http://hamburg.onruby.de/locations/none?locale=en).

If you are unsure when we are still looking for a location have a look at our [schedule overview](https://github.com/rughh/planning/issues/129)

## Who is the person we can contact in case of questions?
Moritz Kröger

## When do you want to host us?
2024-11-13 19:00

## Is the location address still update to date?
On our [website](https://hamburg.onruby.de/locations) we offer directions to the location. Please check if your address is still correct: 
[ ] Yes
[x] No (please comment the correct address below)

## What do you want to offer?
[x] alcohol free beverages
[x] beer (optional)
[x] snacks (optional) 
[x] dinner (pizza, pretzels, etc.) (optional)

Thank you for supporting us!

**Comment by McRip** [comment]
New address is Oberbaumbrücke 1, 20457 Hamburg.

**Comment by JoschkaSchulz** [comment]
Hey @McRip, sorry for the late reply. As I know we are already full of locations this year. Maybe it would be possible to do the first usergroup in 2025?

**Comment by McRip** [comment]
Let's do February then.

**Comment by JoschkaSchulz** [comment]
Also really nice :) I will add it directly after I'm home again :) thanks for the location offer :)

**Comment by JoschkaSchulz** [comment]
Hey @McRip :) It's a while now. Is the Location Offer still valid?

**Comment by McRip** [comment]
Yes, it’s still valid. We are talking about the 12th of February right?
